archiver_test.go

```diff
@@ -67,7 +67,7 @@ func (s *ArchiverSuite) SetupTest() {
 	s.tmpFs, err = fs.Chroot("tmp")
 	s.NoError(err)
 
-	s.tx = rrepository.NewSivaRootedTransactioner(rrepository.NewLocalCopier(s.rootedFs), s.txFs)
+	s.tx = rrepository.NewSivaRootedTransactioner(rrepository.NewLocalCopier(s.rootedFs, 0), s.txFs)
```

Comment:
we don't have any test, with bucketing?

Reply:
It's tested in core-retrieval:

https://github.com/src-d/core-retrieval/blob/master/repository/repository_test.go#L71-L83

Reply:
I'll add a parameter for bucket size to the test and check both cases in borges.